service-workers/service-worker/import-scripts-resource-map.https.html

```diff
@@ -10,14 +10,23 @@
 // script should be stored on the first import and thereafter that stored
 // script should be loaded. The worker asserts that the stored script was
 // loaded; if the assert fails then registration fails.
-promise_test(t => {
-    const scope = 'resources/import-scripts-resource-map';
-    return service_worker_unregister(t, scope)
-      .then(() => {
-          return navigator.serviceWorker.register(
-              'resources/import-scripts-resource-map-worker.js', {scope: scope});
-        })
-      .then(r => r.unregister());
-  }, 'import the same script URL multiple times');
+
+promise_test(async t => {
+  const SCOPE = 'resources/import-scripts-resource-map';
+  const SCRIPT = 'resources/import-scripts-resource-map-worker.js'
+  await service_worker_unregister(t, SCOPE);
+  const registration =
+      await navigator.serviceWorker.register(SCRIPT, {scope: SCOPE});
```

Comment:
Style nit: please use 2-space or 4-space indentation consistently. (Twice in this file.)

Reply:
Done